ca_gazebo/models/april_tags_camera/model.sdf

```diff
@@ -0,0 +1,48 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='april_tags_camera'>
+    <static>true</static>
+    <pose>-1 0 2 0 1 0</pose>
+    <link name='april_camera_link'>
+      <visual name='visual'>
+        <geometry>
+        <mesh>
+          <uri>model://april_tags_camera/meshes/camera.stl</uri>
+          <scale>2.5 2.5 2.5</scale>
```

Comment:
Scale the mesh in an editor and don't apply this scale.

Reply:
Adressed